<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Graphics/Graphics/Artboard 1@64x.png</normaloff>:/Graphics/Graphics/Artboard 1@64x.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="entirePage" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;

</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>184</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="SideGraphics" stretch="0,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="Logo">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Resources.qrc">:/Graphics/Graphics/AP_Icon.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Texts">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Lable_JohnHafteh">
           <property name="styleSheet">
            <string notr="true">font: 800 25pt &quot;Helvetica Now Display&quot;;</string>
           </property>
           <property name="text">
            <string>Jan Hafteh</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="Subtext">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_FirstLine">
             <property name="styleSheet">
              <string notr="true">font: 400 13pt &quot;Helvetica Now Display&quot;;</string>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;The &lt;span style=&quot; font-weight:700;&quot;&gt;most&lt;/span&gt; trusted name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>-1</number>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_secondLine">
             <property name="styleSheet">
              <string notr="true">font: 400 13pt &quot;Helvetica Now Display&quot;;</string>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;among massaging apps&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>-1</number>
             </property>
             <property name="indent">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="Pages">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>500</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 20px;
border-style: solid;
border-width: 2px;
border-color: #9d9d9d;
padding-left: 10px;
font: 12pt &quot;Helvetica Now Text&quot;;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="LoginPage">
       <widget class="QFrame" name="LoginFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>331</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
border-radius: 20px;
border-style: solid;
border-width: 2px;
border-color: #9d9d9d;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>291</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="LoginArea">
          <item>
           <widget class="QLabel" name="Lable_Username">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;
border-width: 0px;
font: 500 12pt &quot;Helvetica Now Display&quot;;
padding-left: 5px;</string>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Username">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Text</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Lable_Password">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Display</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;
border-width: 0px;
font: 500 12pt &quot;Helvetica Now Display&quot;;
padding-left: 5px;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Password">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Text</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SignUpInsteadButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font: 500 12pt &quot;Helvetica Now Display&quot;;
text-decoration: underline;
color:#956cec;


</string>
            </property>
            <property name="text">
             <string>Sign Up Instead</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="LoginButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Display</family>
              <pointsize>18</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#LoginButton 
{
border-radius: 12px;
background: #956cec;
font: 800 18pt &quot;Helvetica Now Display&quot;;
color: white;
}
QPushButton#LoginButton :hover
{
background: #7152b3;
}
QPushButton#LoginButton :pressed 
{
background: white;
border: 2px solid #956cec; 
color: #956cec;
}</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="fd" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="Header">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>65</y>
          <width>90</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 800 24pt &quot;Helvetica Now Display&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SignUpPage">
       <widget class="QFrame" name="SignUpFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>331</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
border-radius: 20px;
border-style: solid;
border-width: 2px;
border-color: #9d9d9d;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>291</width>
           <height>301</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="SignUpArea">
          <item>
           <widget class="QLabel" name="Lable_Username_New">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;
border-width: 0px;
font: 500 12pt &quot;Helvetica Now Display&quot;;
padding-left: 5px;</string>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_UsernameNew">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Text</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>Set a username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Lable_Password_New">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Display</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 0px;
border-width: 0px;
font: 500 12pt &quot;Helvetica Now Display&quot;;
padding-left: 5px;</string>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_PasswordNew">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Text</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>Enter a secure password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_PasswordReenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica Now Text</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>Reenter the password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="LoginInsteadButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
opacity: 1;
font: 500 12pt &quot;Helvetica Now Display&quot;;
text-decoration: underline;
color:#956cec;


</string>
            </property>
            <property name="text">
             <string>LogIn Instead</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="SignUpButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 12px;
background: #956cec;
font: 800 18pt &quot;Helvetica Now Display&quot;;
color: white;
</string>
            </property>
            <property name="text">
             <string>Sign Up</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="Header_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>65</y>
          <width>125</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 800 24pt &quot;Helvetica Now Display&quot;;</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
